hack/verify-vendor-licenses.sh

```diff
@@ -21,15 +21,6 @@ set -o pipefail
 KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 source "${KUBE_ROOT}/hack/lib/init.sh"
 
-readonly branch=${1:-${KUBE_VERIFY_GIT_BRANCH:-master}}
-if ! [[ ${KUBE_FORCE_VERIFY_CHECKS:-} =~ ^[yY]$ ]] && \
```

Comment:
are we just dropping these since the job runs faster now?

Reply:
Yup! This logic was fragile (only worked if you had a full clone of the repo). With the 3-5 min run time, we can just run this 100% of the time.